### PR TITLE
Fix page numbers for empty category

### LINKED ISSUE #300: Add to Category screen refresh for Page # after deleting category.
**Describe the bug**
The Add to Category view from the home screen does not fully refresh after deleting a custom category.

This results in a minor UI issue.

**To Reproduce**
1.) Launch the app
2.) Navigate to Settings > Categories
3.) Create a new custom category.
4.) Navigate to the Home screen.
5.) Tap on the Keyboard icon
6.) Enter Text and press the Save button -> Notice custom category is displayed and Page # is displayed.
6.) Tap on the Back button and tap on the Settings icon
7.) Delete the custom category just created.
8.) Tap on the back button.  Tap on the X to exit Settings View
9.) Press the Save button in the keyboard view.

**Expected behavior**
? Not sure
Designs show an empty state with page numbers
Not sure if it should be there or if it shouldn't?

**Actual Behavior**
The Add to Category view will show page number with empty state as per design.
but if you press the back button and press the save button again, the Page number disappears.

**Screenshots**
Screen Shot 2020-09-11 at 12 56 11 PM
Screen Shot 2020-09-11 at 1 05 23 PM

**Smartphone (please complete the following information):**
 - Device: Pixel C 
 - OS: Android 10 API 30

### PR DESCRIPTION
**Pull request by jollygreenegiant**
Description: 
Hardcodes page numbers to be page 1 of 1 for an empty category, per the design

Fixes #300 

- [ ] Acceptance Criteria satisfied
- [ ] Regression Testing

### CODE REVIEW

File: app/src/main/java/com/willowtree/vocable/presets/AddToCategoryPickerFragment.kt [hunk 1]
```diff
@@ -133,6 +133,8 @@ class AddToCategoryPickerFragment : BaseFragment<FragmentAddToCategoryPickerBind
                     )
                 }
             }
+        } else {
+            binding.categoryPageNumber.text = getString(R.string.phrases_page_number, 1, 1)
```
Comment (jrtb) [inline app/src/main/java/com/willowtree/vocable/presets/AddToCategoryPickerFragment.kt:137]: I don't know what R is, but LGTM ;)

File: app/build.gradle [hunk 2]
```diff
@@ -18,7 +18,7 @@ android {
         applicationId "com.willowtree.vocable"
         minSdkVersion 24
         targetSdkVersion 29
-        versionCode 25
+        versionCode 26
```
Comment (jrtb) [inline app/build.gradle:21]: Making sure this was intentional :)

**Review by jrtb — APPROVED** [review]
One minor question and one double check, otherwise, LGTM! :)

File: app/build.gradle (hunk 2 above)
Comment (jollygreenegiant) [reply app/build.gradle:21]: It is, it's carryover from another PR, just got the commits overlapped a little.  Once the other PR is merged this diff will go away, 26 is the current released version

File: app/src/main/java/com/willowtree/vocable/presets/AddToCategoryPickerFragment.kt (hunk 1 above)
Comment (jollygreenegiant) [reply app/src/main/java/com/willowtree/vocable/presets/AddToCategoryPickerFragment.kt:137]: R is the parent for all of Android's resource files - strings, integers, dimens, colors, etc :)

Comment (jrtb) [reply app/src/main/java/com/willowtree/vocable/presets/AddToCategoryPickerFragment.kt:137]: Nice! Now I know! :)

**Pull request merged**